payments-core/src/main/java/com/stripe/android/view/CardInputListener.kt

```diff
@@ -39,4 +39,11 @@ interface CardInputListener {
      * the user edits the CVC.
      */
     fun onCvcComplete()
+
+    /**
+     * Called when a valid US zip code has been entered.
+     * May be called multiple times, if the user edits the field.
+     * Only works if the [CardWidget] it's used in collects US cards.
```

Comment:
If the postalCodeRequired is also required to be true in addition to the us zip code required, it might be worth noting in this comment?